.circleci/config.yml

```diff
@@ -36,9 +36,7 @@ jobs:
        - restore_cache:
            key: dependency-cache-{{ checksum "build.gradle" }}
        - run:
-           command: ./gradlew assemble testJar
-       - run:
-           command: ./.circleci/publish.sh
+           command: ./gradlew distributeBuild
```

Comment:
Not running tests before the release? distributeBuild does not trigger running the checks.

Reply:
in config.yml the publish job requires (a passing) build, so tests would get run before publication